### ISSUE TITLE
Python 3 compatibility

### ISSUE THREAD

**Issue by jhford** [taskcluster/slugid.py#2]
```
jhford:~/taskcluster/slugid.py $ python3.5
Python 3.5.1 (default, Dec 10 2015, 13:18:29)
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.1.76)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import slugid
>>> slugid.nice()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jhford/taskcluster/slugid.py/slugid/slugid.py", line 42, in nice
    rawBytes = chr(ord(rawBytes[0]) & 0x7f) + rawBytes[1:]  # Ensure slug starts with [A-Za-f]
TypeError: ord() expected string of length 1, but int found
>>>
```

**Comment by jhford** [comment]
```
>>> slugid.v4()
b'-UW2LXLeRcSWXVVZLrIO9w'
>>> slugid.encode(slugid.v4())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jhford/taskcluster/slugid.py/slugid/slugid.py", line 12, in encode
    return base64.urlsafe_b64encode(uuid_.bytes)[:-2] # Drop '==' padding
AttributeError: 'bytes' object has no attribute 'bytes'
>>> slugid.decode(slugid.v4())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jhford/taskcluster/slugid.py/slugid/slugid.py", line 19, in decode
    return uuid.UUID(bytes=base64.urlsafe_b64decode(slug + '==')) # b64 padding
TypeError: can't concat bytes to str
>>>
```

**Comment by petemoore** [comment]
Resolved in #4. Thanks guys!
